.github/workflows/docs-to-gh-pages.yml

```diff
@@ -14,9 +14,11 @@ jobs:
       - name: Setup Hugo
         uses: peaceiris/actions-hugo@v3
         with:
-          hugo-version: '0.110.0'
+          hugo-version: "0.110.0"
           extended: true
 
+      - uses: dw-labs-org/dart-sass-gha@v1
```

Comment:
Thanks for your efforts here to creating this action! Really appreciate it 🙏 
However, I think using an action here is a bit overkill.
We can install it right out of the snap repo:
```yml
- name: Install Dart Sass
  run: sudo snap install dart-sass
```
This is also what the hugo docs say 🙃 So I would follow that advise.

Reply:
That is what I thought... but I couldn't get it to work. Looking back I have a feeling it might have been a path issue so it might be a case of adding:
```yaml
- run: echo "/snap/bin" >> $GITHUB_PATH
```
Which is easier to maintain than an action, although it was good to learn how to do it.
 
If you install hugo with a snap I think dart-sass is included so maybe that's even easier.